.github/workflows/workflow.yml

```diff
@@ -56,15 +56,15 @@ jobs:
         export JORMUNGANDR_BROKER_URL='amqp://guest:guest@rabbitmq:5672//'
         export KRAKEN_RABBITMQ_HOST='rabbitmq'
         ctest --output-on-failure
+    - name: remove useless tests
+      run: |
+        rm -rf tests/mock_kraken kraken/tests ed/tests fare/tests routing/tests calendar/tests
+        rm -rf georef/tests proximity_list/tests time_tables/tests equipment/tests
+        rm -rf ptreferential/tests
     - name: install tyr dependencies
```

Comment:
There's no need for tyr dependencies if docker_test aren't run

Reply:
Yes but It is a partial change. 
I hope, in short times, to reactivate docker test